packages/containers/src/ComponentForm/ComponentForm.component.js

```diff
@@ -42,21 +42,28 @@ export function toJS(immutableObject) {
  * @param value The input value
  */
 export function resolveNameForTitleMap({ schema, properties, value }) {
-	if (schema.titleMap) {
-		// Here we add a field side by side with the value
-		// to keep the title associated to the value
-		const info = schema.titleMap.find(titleMap => titleMap.value === value);
-
-		const parentKey = schema.key.slice();
-		const key = parentKey.pop();
-		const nameKey = `$${key}_name`;
-		const parentValue = getValue(properties, { key: parentKey });
-
-		if (info) {
-			parentValue[nameKey] = info.name;
-		} else {
-			delete parentValue[nameKey];
-		}
+	if (!schema.titleMap) {
+		return;
+	}
+
+	// Here we add a field side by side with the value
+	// to keep the title associated to the value
+	const valueIsArray = Array.isArray(value);
+	const uniformValue = valueIsArray ? value : [value];
```

Comment:
hmm why not const uniformValue = Array.isArray(value) ? value : [value];

Reply:
Because I do the test twice. That's why I have a variable

Reply:
Ho sorry ;)